{{cookiecutter.project_name}}/Pipfile

```diff
@@ -42,6 +42,7 @@ sphinx = "*"
 sphinx-autodoc-typehints = "*"
 sphinxcontrib-napoleon = "*"
 "doc8" = "*"
+pytest-picked = "*"
```

Comment:
Can you please move this line to the same place where all `pytest` plugins are specified?